app/utils/polyfills.ts

```diff
@@ -27,8 +27,8 @@ export async function loadPolyfills() {
  */
 function supportsResizeObserver() {
   return (
-    "ResizeObserver" in global &&
-    "ResizeObserverEntry" in global &&
+    "ResizeObserver" in window &&
+    "ResizeObserverEntry" in window &&
```

Comment:
Is there a benefit to use global in the frontend? I was able to shim global for npm dependencies, but not for our code.

Reply:
I'd imagine this was a copy/paste to be honest